src/MPCC.cpp

```diff
@@ -355,26 +359,28 @@ int pcc_matrix(int m, int n, int p,
     #pragma omp parallel for private (i,k)
     for (i=0; i<m; i++) {
       for (k=0; k<n; k++) {
-        amask[ i*n + k ] = 1.0;
         if (CHECKNA(A[i*n+k])) { 
           amask[i*n + k] = 0.0;
           A[i*n + k] = 0.0; // set A to 0.0 for subsequent calculations of PCC terms
         }else{
+          amask[ i*n + k ] = 1.0;
```

Comment:
This change changes nothing, and will make the code slower, because of the way CPU branch prediction work

while(x){
  a =1
  if(something) a = 0
}

is faster than: 

while(x){
  if(something) a = 0
  else   a =1
}